test/test_aggregate_status.py

```diff
@@ -0,0 +1,20 @@
+import yaml
+from ros_buildfarm.aggregate_status import get_status_description
+
+def test_aggregate_status():
+    D = yaml.load(open('test/aggregate_status_tests.yaml'))
+    match = 0
+    for i, case in enumerate(D['test_cases']):
+        distro = case.get('distro', 'indigo')
+        blacklist = set(map(tuple, case.get('blacklist', [])))
+        status = get_status_description(case['build_status'], D['expected'][distro], blacklist)
+        assert status == case['result']
+        if status == case['result']:
```

Comment:
These two lines don't seem to make much sense. The `else` case below can never be reached?